internal/providers/docker/components/container/build.go

```diff
@@ -238,3 +257,47 @@ func tarBuildContext(w io.WriteCloser, root string) (err error) {
 	}
 	return nil
 }
+
+func createIgnoreMatcher(buildContext string) (*fileutils.PatternMatcher, error) {
+	// Initialize patterns with always ignored files.
+	const ignoreBasename = ".dockerignore"
```

Comment:
I'll note, just as an FYI, that docker also supports ignore files of the format `<dockerfile-name>.dockerignore`. Though you have to enable Buildkit for that to work. It's reasonably widely used and I've used the feature in every non-trivial docker project I've worked on, including the console.

https://stackoverflow.com/a/57774684/1142314
https://github.com/deref/deref/blob/master/registry/build.Dockerfile.dockerignore

Reply:
Do we have a ticket somewhere to support Buildkit builds?